spec/stack.js

```diff
@@ -3,20 +3,55 @@ import chaiChange from 'chai-change'
 import Stack from '../src/stack'
 
 chai.use(chaiChange)
-
 describe('Stack', () => {
-  'use strict'
+    'use strict'
+
+    it('exists', () => {
+        expect(Stack).to.be.a('function')
+    })
+
+    context('push()', () => {
```

Comment:
An additional test is required in order to determine if the correct value was pushed (the current test only checks to see if the size of the stack changed).